### DISCUSSION TITLE
How do I check polling rate of my device?

### DISCUSSION THREAD

**Discussion by alchem1no** [mat1jaczyyy/Keyboard-Inspector#35] [Help]
I bought Sayodevice that has polling rate of 8000 Hz and I want to check if my PC actually "sees" this polling rate. (I connected it to USB 3.0
I followed some tutorial on YouTube that told me to use this program to check polling rate of my device. The interface, however, look different from the video's and there is no option for analyzing polling rate anymore in the options bar.

![image](https://github.com/user-attachments/assets/fc8b13ae-d71f-4962-ac11-db6cd6ee3e68)

So how do I check my polling rate with this program?

**Comment by mat1jaczyyy** [comment]
This issues forum is meant for development of the software. For general usage questions the [Discord](https://discord.gg/kX4cJQH5Zn) is better suited.

That said, the old analyzer we used to have was extremely primitive and one-dimensional. Keyboard behavior simply cannot be boiled down to a single polling rate number, so we now display all the relevant data as a bunch of graphs on-screen for the user to interpret and understand their keyboard behavior themselves. You probably need some fundamental understanding of signal analysis to be able to interpret these accurately.

It's hard to help you with your results directly if you don't share a .kbi file or at the very least a screenshot of the analysis.
